client/my-sites/themes/validate-filters.js

```diff
@@ -1,16 +1,17 @@
 /**
  * External dependencies
  */
-
 import page from 'page';
 
 /**
  * Internal dependencies
  */
-import getThemeFilterStringFromTerm from 'state/selectors/get-theme-filter-string-from-term';
+import {
+	getThemeFilterStringFromTerm,
+	getThemeFilterTerm,
+	getThemeFilterTermFromString,
+} from 'state/themes/selectors';
 
-import getThemeFilterTerm from 'state/selectors/get-theme-filter-term';
-import getThemeFilterTermFromString from 'state/selectors/get-theme-filter-term-from-string';
 import isValidThemeFilterTerm from 'state/selectors/is-valid-theme-filter-term';
```

Comment:
This seems like another selector that could be moved the same way like we did with others.

Reply:
Good catch! Done 🙂 

It will probably take a few passes before we get everything right 😄